submit-web/src/components/Projects/SubmissionTable.tsx

```diff
@@ -27,17 +27,22 @@ export default function SubmissionTable({
         {!headless && (
           <TableHead sx={{ border: 0 }}>
             <TableRow>
-              <TableCell sx={{ color: BCDesignTokens.themeGray70 }}>
+              <TableCell
+                sx={{
+                  color: BCDesignTokens.themeGray70,
+                  width: "30%",
```

Comment:
do we have to manually set the width like this, is there no prop in Table cell that handles this like 
![image](https://github.com/user-attachments/assets/2dd077eb-a8f9-49c3-aeec-f84e865fdcd3)

Reply:
Is there nothing like a column span sort of prop that can be used?